docs/testplan/multi_tier_fec_test.md

```diff
@@ -0,0 +1,23 @@
+# SONiC Switch FEC Test
```

Comment:
Rename the file to match the title. The FEC test cannot be tested in multi-tier network